### PR TITLE
Create UI_UX.md

### PR DESCRIPTION
**Pull request by agichim**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/UI_UX.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: UI/UX
+about: Suggest a UI/UX idea for this project
+title: In order to [achieve something] As a [role] I want [something to be added or
+  changed]
+labels: enhancement
```
Comment (alexsegura) [inline .github/ISSUE_TEMPLATE/UI_UX.md:6]: Maybe this should also add the [UI/UX label](https://github.com/coopcycle/coopcycle-web/issues?q=is%3Aopen+is%3Aissue+label%3AUI%2FUX)?

Comment (agichim) [reply .github/ISSUE_TEMPLATE/UI_UX.md:6]: Yes, that's what I was trying to accomplish, add it as a label :)

Comment (alexsegura) [reply .github/ISSUE_TEMPLATE/UI_UX.md:6]: It already exists 🙂

Comment (agichim) [reply .github/ISSUE_TEMPLATE/UI_UX.md:6]: But I mean I want to be able to add it here with a pink colour. So when people create an issue they can select UX/UI

image

**Pull request closed without merging**